OJS.Workers.ExecutionStrategies/CPlusPlus/CPlusPlusZipFileExecutionStrategy.cs

```diff
@@ -61,7 +58,7 @@ public class CPlusPlusZipFileExecutionStrategy : BaseCompiledCodeExecutionStrate
                 return result;
             }
 
-            var executor = this.CreateExecutor(ProcessExecutorType.Restricted);
+            var executor = this.CreateExecutor(ProcessExecutorType.Standard);
```

Comment:
Is this necessary? `IProcessExecutorFactory` should handle this correctly when run under `Unix`